command/genesis.go

```diff
@@ -54,13 +54,17 @@ func (c *GenesisCommand) Run(args []string) int {
 	var ibftValidators helperFlags.ArrayFlags
 	var ibftValidatorsPrefixPath string
 
+	// dummy flag
+	var dummyConsensus bool
+
 	flags.StringVar(&dataDir, "data-dir", "", "")
 	flags.StringVar(&name, "name", "example", "")
 	flags.Var(&premine, "premine", "")
 	flags.Uint64Var(&chainID, "chainid", 100, "")
 	flags.BoolVar(&ibftConsensus, "ibft", false, "")
 	flags.Var(&ibftValidators, "ibft-validator", "list of ibft validators")
 	flags.StringVar(&ibftValidatorsPrefixPath, "ibft-validators-prefix-path", "", "")
+	flags.BoolVar(&dummyConsensus, "dummy", false, "")
```

Comment:
Could you do instead a flag called "consensus" which you use to select either ibft or dummy? and then
```
ibftConsensus := consensus == "ibft"
```